cmd/tailscaled/tailscaled.go

```diff
@@ -409,7 +411,9 @@ func startIPNServer(ctx context.Context, logf logger.Logf, logID logid.PublicID,
 	// SIGPIPE sometimes gets generated when CLIs disconnect from
 	// tailscaled. The default action is to terminate the process, we
 	// want to keep running.
-	signal.Ignore(syscall.SIGPIPE)
+	if sigPipe != nil {
+		signal.Ignore(sigPipe)
```

Comment:
signal.Ignore takes an argv of signals to ignore, if argv is empty all incoming signals are ignored. The docs don't actually say what nil means. Looks like the meaning varies by platform, on plan9 it ends up signal_ignore'ing `-1`, which I stopped there, but I assume gets rejected somewhere along the way to talking to whoever deals with signal ignores (tty fd?), unless plan9's interface has negative signal numbers?

Reply:
signal.Ignore takes `...os.Signal`. If I passed a nil here, it'd be like passing `[]os.Signal{nil}`, a 1-length slice with a nil interface value.

Reply:
yeah, a nil interface value gets turned into a -1 by signum(), I stopped there, as I remembered that the place that this num is going to be sent isn't as simple as a syscall interface, and I didn't want to go dig. I assume it's fine, but I have no idea